From 638ce45addd6dd9f456699bf0f66cdbec1dc78ab Mon Sep 17 00:00:00 2001
From: Todd Zullinger <tmz@pobox.com>
Date: Mon, 25 Jun 2018 10:28:17 -0400
Subject: [PATCH] Avoid python-3.7 reserved keyword "async"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The "async" keyword was added in python-3.7ยน.  Using it throws
SyntaxError exception.

ยน https://docs.python.org/3.7/whatsnew/3.7.html
  https://bugs.python.org/issue30406
---
 src/gpodder/gtkui/services.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gpodder/gtkui/services.py b/src/gpodder/gtkui/services.py
index e65634e9..69de8553 100644
--- a/src/gpodder/gtkui/services.py
+++ b/src/gpodder/gtkui/services.py
@@ -101,7 +101,7 @@ def replace_cover(self, channel, custom_url=None):
         """
         self.request_cover(channel, custom_url)
 
-    def __get_cover(self, channel, url, async=False, avoid_downloading=False):
+    def __get_cover(self, channel, url, async_mode=False, avoid_downloading=False):
         def get_filename():
             return self.downloader.get_cover(channel.cover_file,
                     url or channel.cover_url, channel.url, channel.title,
@@ -131,7 +131,7 @@ def get_filename():
                 logger.warn('Corrupt cover art on server, deleting', exc_info=True)
                 util.delete_file(filename)
 
-        if async:
+        if async_mode:
             self.notify('cover-available', channel, pixbuf)
         else:
             return (channel.url, pixbuf)
